extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -452,8 +452,10 @@ export class AmpStoryPage extends AMP.BaseElement {
         switch (mediaEl.tagName.toLowerCase()) {
           case 'amp-img':
           case 'amp-anim':
-            mediaEl.signals().whenSignal(CommonSignals.LOAD_END)
-                .then(resolve, resolve);
+            if (mediaEl.signals) {
```

Comment:
When there are new updates, `amp-live-list` does an `importNode(newPage)` for the new `amp-story-page`. (See snippet below this comment)

This triggers the constructor for `AmpStoryPage`, which in turn calls this method (`waitForMediaLayout_`). But `signals` isn't defined yet in the prototype chain for some reason. So it throws `mediaEl.signals is not a function`.

I think there's another way to fix this -- one that fixes it from the root cause, but I wasn't sure it was on the scope for this PR. I can add a TODO here since I think this would require a larger change.
 

https://github.com/ampproject/amphtml/blob/21057a5e553eba9ab1fb2d6f4655599ac25852a1/extensions/amp-live-list/0.1/amp-live-list.js#L687-L689